static/src/uu/index.scss

```diff
@@ -1,7 +1,22 @@
-@import "variables";
 @import "font";
 @import "badge";
 
-img.logo {
-    height: 48px;
-}
+// Dark theme specific settings
+[data-bs-theme="dark"] {
+    /* Breadcrumb */
+    .breadcrumb {
+        --bs-breadcrumb-bg: var(--bs-black);

```

Comment:
Maybe change  `--bs-breadcrumb-bg` from `--bs-black` to `--bs-tertiary-bg`:
![Screenshot 2023-08-18 at 15-31-02 Yoda - Dev ‐ Home](https://github.com/UtrechtUniversity/yoda-portal/assets/29246264/fd753d8a-1a2a-41fb-a20d-7af6e6e51a99)